dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.ts

```diff
@@ -32,7 +24,7 @@ export class TfmDealCancellationRepo {
     }
 
     if (!matchingDeal.tfm.cancellation) {
-      throw new DealCancellationNotFoundError(dealId.toString());
+      return {};
```

Comment:
Why do we want to return an object here - is the cancellation object not created when a deal reaches TFM (and therefore all deals should have a cancellation object)?

Reply:
Nope, the trend in tfm deals seems to be create objects when they're needed (e.g. ammendments) so I've stuck to that.
I think this is the easiest thing to handle from a front end perspective - (I could return a 404 but that raises some difficult questions given the slightly odd error handling in tfm-api/ui. Alternatively `null` or `undefined` would work but I think I prefer an empty object). 

I expect we'll add a `stage` to the cancellation object at some point & then it might make more sense for this to return something like `stage: 'Not started'`

Reply:
Can we please add a line on why we are returning an empty object please (pretty much what you have explained above), for a better DevX.

Reply:
Interesting - not for this PR, but maybe we should create it during TFM deal creation if it's giong to cause some odd behaviours/logic/returns et